src/io.jl

```diff
@@ -47,11 +47,12 @@ write(io, obj::GeoJSONT) = JSON3.write(io, obj)
 write(obj::GeoJSONT) = JSON3.write(obj)
 
 # GeoInterface supported objects
-write(io, obj) = JSON3.write(io, _lower(obj))
-write(obj) = JSON3.write(_lower(obj))
+write(io, obj; geometrycolumn = first(GI.geometrycolumns(obj))) = JSON3.write(io, _lower(obj; geometrycolumn))
```

Comment:
```suggestion
write(io, obj; kw...) = JSON3.write(io, _lower(obj; kw...))
```
It can just be in _lower ?

Reply:
I want this specifically in `write` so errors are nice